### ISSUE TITLE
[MacOS Montery] Timed out launching Ubuntu VMs

### ISSUE THREAD

**Issue by vietng** [canonical/multipass#2720]
Hi there,

Multipass has stopped working on my environment. I am unable to launch any Ubuntu VM (18.04, 20.04 and 22.04).

Error
```
multipass launch 18.04
launch failed: The following errors occurred:
sensuous-turnstone: timed out waiting for response
```

Please see error log below
```
[2022-08-25T09:41:40.982] [debug] [qemu-system-aarch64] [15001] started: qemu-system-aarch64 --version
[2022-08-25T09:43:15.543] [debug] [qemu-img] [15011] started: qemu-img info --output=json /var/root/Library/Caches/multipassd/qemu/vault/images/bionic-20220810/ubuntu-18.04-server-cloudimg-arm64.img
[2022-08-25T09:43:15.553] [debug] [qemu-img] [15012] started: qemu-img info /var/root/Library/Caches/multipassd/qemu/vault/images/bionic-20220810/ubuntu-18.04-server-cloudimg-arm64.img
[2022-08-25T09:43:15.558] [debug] [qemu-img] [15013] started: qemu-img resize /var/root/Library/Application Support/multipassd/qemu/vault/instances/sensuous-turnstone/ubuntu-18.04-server-cloudimg-arm64.img 5368709120
[2022-08-25T09:43:15.566] [debug] [qemu-img] [15014] started: qemu-img snapshot -l /var/root/Library/Application Support/multipassd/qemu/vault/instances/sensuous-turnstone/ubuntu-18.04-server-cloudimg-arm64.img
[2022-08-25T09:43:15.570] [debug] [sensuous-turnstone] process working dir ''
[2022-08-25T09:43:15.570] [info] [sensuous-turnstone] process program 'qemu-system-aarch64'
[2022-08-25T09:43:15.570] [info] [sensuous-turnstone] process arguments '-machine, virt,highmem=off, -accel, hvf, -drive, file=/Library/Application Support/com.canonical.multipass/bin/../Resources/qemu/edk2-aarch64-code.fd,if=pflash,format=raw,readonly=on, -cpu, cortex-a72, -nic, vmnet-shared,model=virtio-net-pci,mac=52:54:00:dc:36:09, -device, virtio-scsi-pci,id=scsi0, -drive, file=/var/root/Library/Application Support/multipassd/qemu/vault/instances/sensuous-turnstone/ubuntu-18.04-server-cloudimg-arm64.img,if=none,format=qcow2,discard=unmap,id=hda, -device, scsi-hd,drive=hda,bus=scsi0.0, -smp, 1, -m, 1024M, -qmp, stdio, -chardev, null,id=char0, -serial, chardev:char0, -nographic, -cdrom, /var/root/Library/Application Support/multipassd/qemu/vault/instances/sensuous-turnstone/cloud-init-config.iso'
[2022-08-25T09:43:15.572] [debug] [qemu-system-aarch64] [15015] started: qemu-system-aarch64 -machine virt,highmem=off -nographic -dump-vmstate /private/var/folders/zz/zyxvpxvq6csfxvn_n0000000000000/T/multipassd.ayOHck
[2022-08-25T09:43:15.601] [info] [sensuous-turnstone] process state changed to Starting
[2022-08-25T09:43:15.603] [info] [sensuous-turnstone] process state changed to Running
[2022-08-25T09:43:15.603] [debug] [qemu-system-aarch64] [15016] started: qemu-system-aarch64 -machine virt,highmem=off -accel hvf -drive file=/Library/Application Support/com.canonical.multipass/bin/../Resources/qemu/edk2-aarch64-code.fd,if=pflash,format=raw,readonly=on -cpu cortex-a72 -nic vmnet-shared,model=virtio-net-pci,mac=52:54:00:dc:36:09 -device virtio-scsi-pci,id=scsi0 -drive file=/var/root/Library/Application Support/multipassd/qemu/vault/instances/sensuous-turnstone/ubuntu-18.04-server-cloudimg-arm64.img,if=none,format=qcow2,discard=unmap,id=hda -device scsi-hd,drive=hda,bus=scsi0.0 -smp 1 -m 1024M -qmp stdio -chardev null,id=char0 -serial chardev:char0 -nographic -cdrom /var/root/Library/Application Support/multipassd/qemu/vault/instances/sensuous-turnstone/cloud-init-config.iso
[2022-08-25T09:43:15.603] [info] [sensuous-turnstone] process started
[2022-08-25T09:43:15.603] [debug] [sensuous-turnstone] Waiting for SSH to be up
[2022-08-25T09:43:15.639] [debug] [sensuous-turnstone] QMP: {"QMP": {"version": {"qemu": {"micro": 0, "minor": 2, "major": 6}, "package": ""}, "capabilities": ["oob"]}}

[2022-08-25T09:43:15.658] [debug] [sensuous-turnstone] QMP: {"return": {}}

[2022-08-25T09:43:37.837] [debug] [sensuous-turnstone] QMP: {"timestamp": {"seconds": 1661391817, "microseconds": 837213}, "event": "NIC_RX_FILTER_CHANGED", "data": {"path": "/machine/unattached/device[6]/virtio-backend"}}
```

Multipass info
```
multipass   1.10.1+mac
multipassd  1.10.1+mac
```

MacOs version
```
ProductName:	macOS
ProductVersion:	12.5.1
BuildVersion:	21G83
```

I noticed that there is a line `Waiting for SSH to be up` in the log. Not sure if that is the culprit.

Things I have tried:
* Reinstall multipass (using brew)
* Add my user to the wheel group which have read/write permission to `/var/run/multipass_socket`
* Have a coffee and restart my laptop :)

Thanks.

**Comment by gavinbenda** [comment]
Getting the same here (and same setup as you).

**Comment by wesreisz** [comment]
Running into what looks like the same issue. Worked quite a bit with multicluster k8s instances. Think I ran into a resource constraint causing my whole box to freeze. On reboot, multipass instances were in an unknown state. Deleted them and recreated them, all attempts hang in unknown. I've uninstalled from the `uninstall.sh` script and using brew without success. 

Logs don't appear to me at least to indicate anything (though, I'm not sure I know what I'm looking at). I'm at a loss. Any suggestions?

multipass launch bionic --name test                           
launch failed: The following errors occurred:                                   
test: timed out waiting for response

multipass version                       
multipass   1.10.1+mac
multipassd  1.10.1+mac

Logs:
[2022-08-30T14:48:42.112] [debug] [update] Latest Multipass release available is version 1.10.1
[2022-08-30T14:48:43.519] [info] [VMImageHost] Did not find any supported products in "appliance"
[2022-08-30T14:48:43.548] [info] [rpc] gRPC listening on unix:/var/run/multipass_socket
[2022-08-30T14:48:43.549] [warning] [Qt] QIODevice::write (QFile, "/var/root/Library/Caches/multipassd/qemu/vault/multipassd-image-records.json"): device not open
[2022-08-30T14:48:43.549] [info] [daemon] Starting Multipass 1.10.1+mac
[2022-08-30T14:48:43.549] [info] [daemon] Daemon arguments: /Library/Application Support/com.canonical.multipass/bin/multipassd --verbosity debug
[2022-08-30T14:49:09.603] [debug] [qemu-system-aarch64] [3591] started: qemu-system-aarch64 --version
[2022-08-30T14:49:28.216] [debug] [qemu-img] [3614] started: qemu-img info --output=json /var/root/Library/Caches/multipassd/qemu/vault/images/bionic-20220810/ubuntu-18.04-server-cloudimg-arm64.img
[2022-08-30T14:49:28.469] [debug] [qemu-img] [3615] started: qemu-img info /var/root/Library/Caches/multipassd/qemu/vault/images/bionic-20220810/ubuntu-18.04-server-cloudimg-arm64.img
[2022-08-30T14:49:28.477] [debug] [qemu-img] [3616] started: qemu-img resize /var/root/Library/Application Support/multipassd/qemu/vault/instances/apt-whitethroat/ubuntu-18.04-server-cloudimg-arm64.img 5368709120
[2022-08-30T14:49:28.485] [debug] [qemu-img] [3617] started: qemu-img snapshot -l /var/root/Library/Application Support/multipassd/qemu/vault/instances/apt-whitethroat/ubuntu-18.04-server-cloudimg-arm64.img
[2022-08-30T14:49:28.490] [debug] [apt-whitethroat] process working dir ''
[2022-08-30T14:49:28.490] [info] [apt-whitethroat] process program 'qemu-system-aarch64'
[2022-08-30T14:49:28.490] [info] [apt-whitethroat] process arguments '-machine, virt,highmem=off, -accel, hvf, -drive, file=/Library/Application Support/com.canonical.multipass/bin/../Resources/qemu/edk2-aarch64-code.fd,if=pflash,format=raw,readonly=on, -cpu, cortex-a72, -nic, vmnet-shared,model=virtio-net-pci,mac=52:54:00:ef:5b:cf, -device, virtio-scsi-pci,id=scsi0, -drive, file=/var/root/Library/Application Support/multipassd/qemu/vault/instances/apt-whitethroat/ubuntu-18.04-server-cloudimg-arm64.img,if=none,format=qcow2,discard=unmap,id=hda, -device, scsi-hd,drive=hda,bus=scsi0.0, -smp, 1, -m, 1024M, -qmp, stdio, -chardev, null,id=char0, -serial, chardev:char0, -nographic, -cdrom, /var/root/Library/Application Support/multipassd/qemu/vault/instances/apt-whitethroat/cloud-init-config.iso'
[2022-08-30T14:49:28.492] [debug] [qemu-system-aarch64] [3618] started: qemu-system-aarch64 -machine virt,highmem=off -nographic -dump-vmstate /private/var/folders/zz/zyxvpxvq6csfxvn_n0000000000000/T/multipassd.Jiokat
[2022-08-30T14:49:28.522] [info] [apt-whitethroat] process state changed to Starting
[2022-08-30T14:49:28.524] [info] [apt-whitethroat] process state changed to Running
[2022-08-30T14:49:28.524] [debug] [qemu-system-aarch64] [3619] started: qemu-system-aarch64 -machine virt,highmem=off -accel hvf -drive file=/Library/Application Support/com.canonical.multipass/bin/../Resources/qemu/edk2-aarch64-code.fd,if=pflash,format=raw,readonly=on -cpu cortex-a72 -nic vmnet-shared,model=virtio-net-pci,mac=52:54:00:ef:5b:cf -device virtio-scsi-pci,id=scsi0 -drive file=/var/root/Library/Application Support/multipassd/qemu/vault/instances/apt-whitethroat/ubuntu-18.04-server-cloudimg-arm64.img,if=none,format=qcow2,discard=unmap,id=hda -device scsi-hd,drive=hda,bus=scsi0.0 -smp 1 -m 1024M -qmp stdio -chardev null,id=char0 -serial chardev:char0 -nographic -cdrom /var/root/Library/Application Support/multipassd/qemu/vault/instances/apt-whitethroat/cloud-init-config.iso
[2022-08-30T14:49:28.524] [info] [apt-whitethroat] process started
[2022-08-30T14:49:28.524] [debug] [apt-whitethroat] Waiting for SSH to be up
[2022-08-30T14:49:28.855] [debug] [apt-whitethroat] QMP: {"QMP": {"version": {"qemu": {"micro": 0, "minor": 2, "major": 6}, "package": ""}, "capabilities": ["oob"]}}

[2022-08-30T14:49:28.881] [debug] [apt-whitethroat] QMP: {"return": {}}

[2022-08-30T14:49:51.061] [debug] [apt-whitethroat] QMP: {"timestamp": {"seconds": 1661885391, "microseconds": 61241}, "event": "NIC_RX_FILTER_CHANGED", "data": {"path": "/machine/unattached/device[6]/virtio-backend"}}

[2022-08-30T15:00:43.234] [debug] [sshfs-mounts] No mounts to stop for instance "apt-whitethroat"
[2022-08-30T15:00:43.235] [debug] [apt-whitethroat] QMP: {"return": {}}

[2022-08-30T15:00:43.235] [debug] [apt-whitethroat] QMP: {"timestamp": {"seconds": 1661886043, "microseconds": 235117}, "event": "POWERDOWN"}

[2022-08-30T15:00:43.235] [info] [apt-whitethroat] VM powering down
[2022-08-30T15:00:43.500] [debug] [apt-whitethroat] QMP: {"timestamp": {"seconds": 1661886043, "microseconds": 500692}, "event": "SHUTDOWN", "data": {"guest": true, "reason": "guest-shutdown"}}

[2022-08-30T15:00:43.500] [info] [apt-whitethroat] VM shut down
[2022-08-30T15:00:43.647] [info] [apt-whitethroat] process state changed to NotRunning
[2022-08-30T15:00:43.649] [info] [apt-whitethroat] process finished with exit code 0
[2022-08-30T15:01:00.060] [info] [VMImageHost] Did not find any supported products in "appliance"
[2022-08-30T15:01:00.062] [debug] [qemu-system-aarch64] [4725] started: qemu-system-aarch64 --version
[2022-08-30T15:01:00.079] [debug] [qemu-img] [4726] started: qemu-img info /var/root/Library/Caches/multipassd/qemu/vault/images/bionic-20220810/ubuntu-18.04-server-cloudimg-arm64.img
[2022-08-30T15:01:00.141] [debug] [qemu-img] [4727] started: qemu-img resize /var/root/Library/Application Support/multipassd/qemu/vault/instances/test/ubuntu-18.04-server-cloudimg-arm64.img 5368709120
[2022-08-30T15:01:00.149] [debug] [qemu-img] [4728] started: qemu-img snapshot -l /var/root/Library/Application Support/multipassd/qemu/vault/instances/test/ubuntu-18.04-server-cloudimg-arm64.img
[2022-08-30T15:01:00.155] [debug] [test] process working dir ''
[2022-08-30T15:01:00.155] [info] [test] process program 'qemu-system-aarch64'
[2022-08-30T15:01:00.155] [info] [test] process arguments '-machine, virt,highmem=off, -accel, hvf, -drive, file=/Library/Application Support/com.canonical.multipass/bin/../Resources/qemu/edk2-aarch64-code.fd,if=pflash,format=raw,readonly=on, -cpu, cortex-a72, -nic, vmnet-shared,model=virtio-net-pci,mac=52:54:00:70:3b:36, -device, virtio-scsi-pci,id=scsi0, -drive, file=/var/root/Library/Application Support/multipassd/qemu/vault/instances/test/ubuntu-18.04-server-cloudimg-arm64.img,if=none,format=qcow2,discard=unmap,id=hda, -device, scsi-hd,drive=hda,bus=scsi0.0, -smp, 1, -m, 1024M, -qmp, stdio, -chardev, null,id=char0, -serial, chardev:char0, -nographic, -cdrom, /var/root/Library/Application Support/multipassd/qemu/vault/instances/test/cloud-init-config.iso'
[2022-08-30T15:01:00.157] [debug] [qemu-system-aarch64] [4729] started: qemu-system-aarch64 -machine virt,highmem=off -nographic -dump-vmstate /private/var/folders/zz/zyxvpxvq6csfxvn_n0000000000000/T/multipassd.oKPrBT
[2022-08-30T15:01:00.188] [info] [test] process state changed to Starting
[2022-08-30T15:01:00.190] [info] [test] process state changed to Running
[2022-08-30T15:01:00.190] [debug] [qemu-system-aarch64] [4730] started: qemu-system-aarch64 -machine virt,highmem=off -accel hvf -drive file=/Library/Application Support/com.canonical.multipass/bin/../Resources/qemu/edk2-aarch64-code.fd,if=pflash,format=raw,readonly=on -cpu cortex-a72 -nic vmnet-shared,model=virtio-net-pci,mac=52:54:00:70:3b:36 -device virtio-scsi-pci,id=scsi0 -drive file=/var/root/Library/Application Support/multipassd/qemu/vault/instances/test/ubuntu-18.04-server-cloudimg-arm64.img,if=none,format=qcow2,discard=unmap,id=hda -device scsi-hd,drive=hda,bus=scsi0.0 -smp 1 -m 1024M -qmp stdio -chardev null,id=char0 -serial chardev:char0 -nographic -cdrom /var/root/Library/Application Support/multipassd/qemu/vault/instances/test/cloud-init-config.iso
[2022-08-30T15:01:00.190] [info] [test] process started
[2022-08-30T15:01:00.190] [debug] [test] Waiting for SSH to be up
[2022-08-30T15:01:00.293] [debug] [test] QMP: {"QMP": {"version": {"qemu": {"micro": 0, "minor": 2, "major": 6}, "package": ""}, "capabilities": ["oob"]}}

[2022-08-30T15:01:00.315] [debug] [test] QMP: {"return": {}}

[2022-08-30T15:01:22.599] [debug] [test] QMP: {"timestamp": {"seconds": 1661886082, "microseconds": 599544}, "event": "NIC_RX_FILTER_CHANGED", "data": {"path": "/machine/unattached/device[6]/virtio-backend"}}

**Comment by townsend2010** [comment]
Hey all,

These issues look to be the dreaded [macOS firewall issue](https://github.com/canonical/multipass/issues/2387).  I'm duplicating this to that bug.

Duplicate of #2387
